.github/workflows/ci.yaml

```
@@ -36,6 +36,22 @@ jobs:
           pip install gersemi
           gersemi --check --diff .
 
+  Non-Default-Build-Config:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/hrzlgnm/monkas-github-gcc:v1@sha256:e53c157d3aabd78259a3653790e68e839e8033907c224239f9933db647e34868
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - name: Run sccache-action
+        if: ${{ matrix.toolchain.sccache }}
```

actionlint (1.7.7)

property "toolchain" is not defined in object type {}

(expression)